examples/java/src/main/java/org/apache/beam/examples/complete/TfIdf.java

```diff
@@ -209,7 +209,7 @@ public ComputeTfIdf() { }
       final PCollectionView<Long> totalDocuments =
           uriToContent
           .apply("GetURIs", Keys.<URI>create())
-          .apply("RemoveDuplicateDocs", RemoveDuplicates.<URI>create())
+          .apply("RemoveDuplicateDocs", Distinct.<URI>create())
```

Comment:
Thanks @eljefe6a. I do see RemoveDuplicateTimestamps as well. Best to have that changed as well.
Posting a commit.